alibi_detect/od/_knn.py

```diff
@@ -1,10 +1,17 @@
+"""
+.. automodule:: noodle
+   :members:
+   :private-members:
+"""
```

Comment:
Is this to look at docstrings for a private module?

Reply:
Ahh, I put it there to check the docstrings :man_facepalming:, I've removed it!